awx/main/access.py

```diff
@@ -1997,7 +1998,15 @@ class WorkflowJobNodeAccess(BaseAccess):
     )
 
     def filtered_queryset(self):
-        return self.model.objects.filter(workflow_job__unified_job_template__in=UnifiedJobTemplate.accessible_pk_qs(self.user, 'read_role'))
+        return self.model.objects.filter(
+            Q(workflow_job__unified_job_template__in=UnifiedJobTemplate.accessible_pk_qs(self.user, 'read_role'))
+            | Q(workflow_job__organization__in=Organization.objects.filter(Q(admin_role__members=self.user)))
+        )
+
+    def can_read(self, obj):
```

Comment:
```suggestion
    def can_read(self, obj):
        """Overriding this opens up detail view access for users that launched a bulk job. Otherwise it would not be visible since it has no workflow job template."""
```